ci-operator/step-registry/rhscl/deploy-and-test/OWNERS

```diff
@@ -0,0 +1,12 @@
+# Fetched from https://github.com/rhpit/interop-tests root OWNERS
+# If the repo had OWNERS_ALIASES then the aliases were expanded
+# Logins who are not members of 'openshift' organization were filtered out
+# See the OWNERS docs: https://git.k8s.io/community/contributors/guide/owners.md
+
+approvers:
+- cspi-qe-ocp-lp
+- phracek
+options: {}
+reviewers:
```

Comment:
Do you want to add one of the other owners. I see in the config there is also `aslicerh`

Reply:
owners you mentioned are not related to rhscl, probably for dotnet